requirements.txt

```diff
@@ -1,3 +1,4 @@
+urllib3
```

Comment:
should we need the specific version for the urllib3 package?

Reply:
Sorry Coulson, I make a mistake to close your comment